Ocean/README.md

```diff
@@ -0,0 +1,59 @@
+# Polar Datacube Generation
+
+Information dedicated to the generation of the Ocean Datacubes. In this folder two ocean data cubes were created: 1) The Particulate Organic Carbon (POC) Data Cube at 4 km and 2) the Oceanic Export Production, Phytoplankton Carbon and Primary Production Data Cube at 9 km.
+
+## Cubes generation process
+
+The cubes generation process is divided in four phases:
+
+### 1. Downloading required raw datasets
+
+Datasets were downloaded via the source information provided in `cube4km.geojson` and `cube9km.geojson`. Datasets were downloaded in `.nc` format.
+
+To download the datasets, it is required to run the following files:
```

Comment:
Please add a description, that deep-esdl-cube-gen python env needs to be used for executing the scripts. And how to do this.